recipes/default.rb

```diff
@@ -17,5 +17,22 @@
 # limitations under the License.
 #
 
+# if trying to use opcache with an incompatible version of php (default pkg)
+# and using an unsupported version of php. use_ius must be true to install a valid php interpreter.
+if node['osl-php']['use_opcache'] && node['php']['version'].to_f < 5.5
+  raise 'Must use PHP >= 5.5 with ius enabled to use Zend Opcache.  Try adding '\
+        "'node.default['osl-php']['use_ius'] = true' and install a proper version of php."
+end
+
+node.normal['osl-php']['php_packages'] += %w(opcache) if node['osl-php']['use_opcache']
+
 include_recipe 'osl-php::packages'
 include_recipe 'php::default'
+
+if node['osl-php']['use_opcache']
+  template '/etc/php.d/10-opcache.ini' do
+    source 'opcache.ini.erb'
+    mode '0444'
+    variables(options: node['osl-php']['opcache'])
+  end
+end
```

Comment:
I've been looking at this and I can see this being some code duplication down the line. I think what might be better is to create a ``php_ini`` custom resource in another PR which manages ``/etc/php.d/`` files. Ideally, it should just take a hash as a parameter like you're doing now, but it would be more generic. Then we could just convert this recipe to use this.